meta.yaml

```diff
@@ -74,6 +74,7 @@ requirements:
     # Multi language support:
     - ncl>=6.5.0
     - r-base
+    - jinja2
```

Comment:
Can you please move this to the `# Normally installed via pip:` section above and maintain alphabetical ordering?